src/app-bar.jsx

```diff
@@ -105,16 +105,22 @@ const AppBar = React.createClass({
 
     /**
      * Callback function for when the left icon is selected via a touch tap.
+     *
+     * @param {object} event TouchTap event targeting the left `IconButton`.
```

Comment:
oh, well anyway it doesn't matter, either way we'll have to come up with better conventions anyway. I mean these aren't real events anyway they are SyntheticEvents that React creates. I wouldn't bother anyway. we'll have to discuss this further. keep it as it is for now :+1: